src/api/API.js

```diff
@@ -108,8 +74,11 @@ const API = {
    * @throws {APIError} On server error.
    * @returns {Promise<UserResponse>}
    */
-  getInfo: async (token, id) =>
-    fetch(relURL(`/users/${id}`), getOptions(token)).then(handleResponse),
+  async getInfo(token, id) {
+    return axios.get(`/users/${id}`, {
+      headers: { Authorization: token },
+    });
```

Comment:
I think you might be missing a `.then(response => response.data)` to return the data from the response